app/controllers/locations_controller.rb

```diff
@@ -1,8 +1,8 @@
 class LocationsController < ApplicationController
-  def create
-    @location = build_location
+  def update
+    @location = find_location
```

Comment:
modified in https://github.com/thoughtbot/freshfoodconnect/pull/8/commits/0743dcc47c6d6442943e8b1b1a8147b930108883